packages/hash/frontend/src/shared/layout/layout-with-sidebar/notification-banner.tsx

```diff
@@ -0,0 +1,48 @@
+import { VoidFunctionComponent } from "react";
+import { Box } from "@mui/material";
+import { Button } from "../../ui/button";
+import { useArchivePage } from "../../../components/hooks/useArchivePage";
+import { useRouteAccountInfo, useRoutePageInfo } from "../../routing";
+
+export const PageNotificationBanner: VoidFunctionComponent = () => {
+  const { accountId } = useRouteAccountInfo();
+  const { pageEntityId } = useRoutePageInfo();
+
+  const { unarchivePage } = useArchivePage(accountId, pageEntityId);
+
+  return (
+    <Box
+      sx={({ palette }) => ({
+        color: palette.common.white,
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        width: 1,
+        background: "#EB5757",
```

Comment:
Please avoid hard-coding colors in files. We have red 50/60 theme colors that would work here. See https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-Editor?node-id=828%3A67111

Reply:
Addressed in f2ca1bf4a3b685a8d9e072bb9550dff18db6e0ed